src/blog/2025-09-19-oss-summit-europe-2025.md

```diff
@@ -0,0 +1,101 @@
+* **Unsustainable motions.** Travel-heavy schedules burn people out and waste money. Replace a portion of travel with scalable digital programs, local community partners, and content that compounds over time. When you do travel, tie every trip to a measurable goal.
+
+The Foundation will publish playbooks to facilitate cross-functional collaboration and promote sustainable operating models. These will include checklists, templates for shared dashboards, and example rhythms for data reviews involving product, marketing, and sales teams.
+
+### Which skills matter most now?
+
+This is where the craft of DevRel meets the discipline of product.
+
+* **Authentic, developer-first communication.** Speak to the problems developers actually have. Show how to solve them with real code, real examples, and real constraints.  
+* **Empathy and persona awareness.** Know who you are helping. A senior platform engineer has different constraints than a startup founder. Route your work accordingly.  
+* **Technical advocacy or strong routing to technical owners.** You do not need to be the deepest expert in every stack. You do need to get developers unblocked quickly. Either do it yourself or build the internal routing that makes it happen.  
+* **Data literacy.** You do not need to be a data scientist. You do need to choose the right metrics, interpret what they mean, and tell a clear business story. Learn enough SQL or analytics tooling to answer basic questions yourself. Pair with data partners for the rest.
+
+These skills are learnable. The Foundation will gather examples and training paths you can use to level up your team.
+
+## How you can participate
+
+Foundations are only as strong as the people who shape them. Here is how to plug in.
+
+1. [**Join our Discord community**](https://discord.gg/G7CSTKZcuT) **and introduce yourself.** Share what you work on, your product context, and what you want from the Foundation.  
+2. [**Explore working groups**](https://dev-rel.org/about/working-groups)**.** Pick one that fits your skills and goals. Personas, metrics, community operations, or another area you care about.  
+3. **Contribute to the [Persona Library](https://github.com/DevRel-Foundation/persona-library) and [Metrics efforts](https://github.com/DevRel-Foundation/metrics-index).** Bring your real-world examples. If your team already uses a persona or a metric definition that works, contribute it so others can adapt it.  
+4. **Share challenges and wins from your org.** We will not publish anything without your consent, but your experience can become a pattern others can use.  
+5. **Pilot a playbook.** Take one of the early frameworks, put it to work for a quarter, and share what you learned. We will iterate together.
+
+You do not need permission to start. If you see a gap, propose a contribution. If you have a question, ask it in public. If you can help another team avoid a month of trial and error, share your approach.
+
```

Comment:
I'd suggest either not including this section since it duplicates what is listed elsewhere and/or is incomplete and has a chance to drift (the discord invite link is biting us for example). This is a partial list of projects so also sets wrong expectations.

Perhaps just link or reference https://dev-rel.org/join to stay on message. If there is something here that is missing from that page, let's update that page.